internal/pkg/cli/svc_deploy.go

```diff
@@ -550,6 +583,11 @@ func (o *deploySvcOpts) stackConfiguration(addonsURL string) (cloudformation.Sta
 		}
 
 		var opts []stack.LoadBalancedWebServiceOption
+
+		// TODO:
+		// 1. Should error out if `nlb.alias` is specified with targetApp.Domain == ""
+		// 2. Should validate `nlb.alias` is specified
+		// 3. ALB block should not be executed if http is disabled
```

Comment:
oh i don't remember did we decide on NLB and ALB being mutually exclusive?

Reply:
No but they should be able to disable http by `http: fault`

Reply:
oh gotcha gotcha, I misunderstood the comment 😅 thank you!